src/main/java/com/epam/tc/hw7/entities/MetalsColorsFormData.java

```diff
@@ -0,0 +1,46 @@
+package com.epam.tc.hw7.entities;
+
+public class MetalsColorsFormData {
+    public String vegetablesSelectedByDefault;
+    public String customRadioOdd;
```

Comment:
Why you use access modifiers public for this fields? You also create methods (setters) with public access modifiers. Change public -> private for fields